### ISSUE TITLE
Struggling to find straightforward API docs for `Model` and `Program` (or any `engine` docs)

### ISSUE THREAD

**Issue by micahscopes** [visgl/luma.gl#1636]
#### Actual Result
For the life of me I can't find the API specs for `Model` or other engine module exports
#### Expected Result
I want a straightforward directory of all the classes and their APIs
#### Reproduce Steps
I've been clicking around the documentation page.  I clicked through every single item in the various "API" sections and used "control-F" to search for "Model".

I finally found it by using google.  It's [here](https://luma.gl/docs/api-reference/engine/model) but I had a hard time finding it in the menu.

One more thing, on my phone the docs page sometimes gets stuck with a full screen length of whitespace and I have to scroll down or refresh to see the docs.  It's also hard to search and navigate from the phone.  There's no way to click through, no clear sequence for API docs.  Sometimes I like using the phone to browse documentation so I can get away from the computer.

![image](https://user-images.githubusercontent.com/389782/160715993-298aa51b-b0f8-4f87-a2b8-c70e0a8490dc.png)
![image](https://user-images.githubusercontent.com/389782/160716012-45d587ae-0995-42f2-8cde-98afc356becd.png)
![image](https://user-images.githubusercontent.com/389782/160716068-208629b6-31b9-4a61-80ef-ddfd1053ca1e.png)

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Test

#### To Do List 
  - [ ] Add label and assign to milestone 
  - [ ] Coding
  - [ ] Doc update
  - [ ] Whats new update 
  - [ ] Test

**Comment by micahscopes** [comment]
Ooh okay I just found the `engine` docs here under "submodule reference":
![image](https://user-images.githubusercontent.com/389782/160716559-99ea0575-c74a-4d6e-9f15-13badebb709c.png)

From the point of view of a newcomer, this is pretty confusing because `Model` is used in a lot of the tutorial examples yet when I go clicking on the various "API" items in the docs it's nowhere to be found.

**Comment by ibgreen** [comment]
@micahscopes Thanks for your feedback! 

My apologies, the docs are in an intermediate state right now as a result of the recent big effort on the new v9 API and WebGPU support. The structure needs another pass. I am hoping to get back to this soon.

And as you say, Model is really the central class in the luma.gl API and it should be very easy to find it.

Also noting that we do have a search function in the header and it does find the Model page.

<img width="1345" alt="Screen Shot 2022-03-29 at 4 28 13 PM" src="https://user-images.githubusercontent.com/7025232/160722676-1b4ac0f4-90dc-4b5e-be9b-fca9e6b5f011.png">

**Comment by ibgreen** [comment]
> One more thing, on my phone the docs page sometimes gets stuck with a full screen length of whitespace and I have to scroll down or refresh to see the docs. It's also hard to search and navigate from the phone. There's no way to click through, no clear sequence for API docs. Sometimes I like using the phone to browse documentation so I can get away from the computer.

Yes better mobile support would be nice. @mletic has been experimenting with moving the docs from gatsby to docusaurus which seemed to work really well on phones.

**Comment by micahscopes** [comment]
> Also noting that we do have a search function in the header and it does find the Model page.
Oh wow I totally missed it :sweat_smile: :mag: :detective:
